src/storage/src/source/kafka.rs

```diff
@@ -552,6 +576,11 @@ impl SourceRender for KafkaSourceConnection {
                                 // watermark. This jump of the progress capability ensures that
                                 // everything until the high watermark will be reclocked to a
                                 // single point.
+                                //
+                                // Note that we expect NEW partitions after the first snapshotting
+                                // of the source to be small, so we don't update snapshot
+                                // metrics when we discover them, like we do with pg/mysql
+                                // tables being added.
```

Comment:
This comment confused me initially because it relates the partitions of a topic with the tables added in mysql/pg. It's as if it's saying that "if we knew new tables were also small we could skip them from snapshot metrics in mysql/pg but because we don't know that we include them". That's not the reason though, the reason is that here the object that we're reading is the **topic** and in mysql/pg the object that we're reading is a **table**. Saying nothing is probably less confusing so I would suggest removing it

Reply:
ill remove it!